exercise-1.js

```diff
@@ -208,9 +315,24 @@
         */
 
         //your code...
-
-
-
+        const jorgeRios = {
+            name:'Jorge Rios',
+            age:29,
+            gender:'m',
+            healthy:'sure',
+            hobbies:'making music',
+            profession:'student',
+            education:'some college',
+            learn: function() {
+                console.log(`${this.name} is learning JavaScript`);
+            },
+            pastTime: function() {
+                console.log(`One of my hobbies is ${this.hobbies} `);
+            }
+        }
```

Comment:
This certainly behaves as expected, and results in an object with the properties expected.  However, the expectation was for the functions to be added to the object _**after**_ the object is created.  How might this be refactored to better fulfill that expectation?